Changelog.md

```diff
@@ -1,5 +1,36 @@
 ## Ignition Msgs 8.x
 
+### Ignition Msgs 8.3.0 (2022-03-24)
+1. backport NavSat message for bridge compatibility
+    * [Pull request #231](https://github.com/ignitionrobotics/ign-msgs/pull/231)
+
+1. Add docs to wheel_slip_parameters_cmd.proto and update entity field
+    * [Pull request #227](https://github.com/ignitionrobotics/ign-msgs/pull/227)
```

Comment:
this can be grouped with the other wheel slip PR, as was done in the ign-msgs7 changelog:

* https://github.com/ignitionrobotics/ign-msgs/compare/ignition-msgs8_8.2.0...aditya/8.3.0#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R79-R81

matching merged forward changelog entries is generally good practice

Reply:
Done